README.md

```diff
@@ -1,12 +1,26 @@
 # Vulcanomicon
 
-The Vulcan Academy's operating system
+The Vulcan Academy's operating system.
 
-1. The goal of the game is to create a learning organization
-1. All players must unanimously agree to all rule changes
-1. Proposals may add, amend or repeal a rule
-1. All rules should be logically self-consistent 
+1. The goal of the game is to create a learning organization.
+1. All players must unanimously agree to all rule changes.
+1. Proposals may add, amend or repeal a rule.
+1. All rules should be logically self-consistent. 
+
+## Reality
+1. Our understanding of __Reality__ is throttled by our ability to percieve it. 
+    * We use logic to describe our understanding of __Reality__.
+1. __Reality__ is exclusively inhabited by __Information__, and patterns thereof.
+    * __Information__ contained within __Empirical Reality__ inhabits 3 spatial dimensions and 1 monotonically increasing temporal dimension.
+1. __Information__ may interact with and modify other __Information__. This is an __Action__.
 
 ## Lexicon
 
-agent: any system that is best characterized as acting according to beliefs
\ No newline at end of file
+* __Reality__: The set of everything that exists or does not exist.
+* __Information__: What is conveyed by a particuar arrangement of things.
```

Comment:
I think this definition could be better.

Reply:
yeah, I'm not sure how to improve it yet but this definition is somewhat confusing or too ambiguous for me

Reply:
How about [__Data__](https://en.wikipedia.org/wiki/Data)?